content/content-tool/tool/src/bundle/helper_eu.properties

```diff
@@ -133,3 +133,29 @@ gen.copycomf=Berretsi copyright-a
 viewing=Bistaratzen
 of=of
 items=itemetatik
+
+att.resource=Select from resources
```

Comment:
Hey @pablosanroman Could you contribute these translations please? thank you

Reply:
Hi @mpellicer !
att.resource=Aukeratu baliabideetatik

Reply:
Thanks Pablo,updated